libs/analysis/aframe/analysis/ledger/injections.py

```diff
@@ -235,7 +249,8 @@ class InterferometerResponseSet(
     InjectionMetadata, ExtrinsicParameterSet, IntrinsicParameterSet
 ):
     def __post_init__(self):
-        InjectionMetadata.__post_init__(self)
+        # initiate chain of mro __post_init__ calls
```

Comment:
What does "mro" mean?

Reply:
method resolution order. When you have a class that inherits from multiple parents, the `super()` call is ambiguous. So in general, calling `super()` refers to the next class in the method resolution order. In this case the order looks like
```
print(InterferometerResponseSet.__mro__)

(aframe.analysis.ledger.injections.InterferometerResponseSet,
 aframe.analysis.ledger.injections.InjectionMetadata,
 aframe.analysis.ledger.injections.ExtrinsicParameterSet,
 aframe.analysis.ledger.injections.EventParameterSet,
 aframe.analysis.ledger.injections.SkyLocationParameterSet,
 aframe.analysis.ledger.injections.IntrinsicParameterSet,
 aframe.analysis.ledger.ledger.Ledger,
 object)

```
(basically it goes left to right through the inheritance tree). 

So every time super() is called, throughout a class hierarcy, it refers to the next class in line. We can ensure that `Ledgers`  `__post_init__` is eventually called by having super().__post_init__ called in each of these classes

Reply:
Ah neat, okay